mk/Makefile.deb

```diff
@@ -52,11 +52,9 @@ SOURCES += $(SOURCEDIR)/xe-linux-distribution
 SOURCES += $(SOURCEDIR)/xe-daemon
 SOURCES += $(SOURCEDIR)/xenstore
 SOURCES += $(SOURCEDIR)/LICENSE
-SOURCES += $(SOURCEDIR)/citrix.list
 SOURCES += $(SOURCEDIR)/xen-vcpu-hotplug.rules
 SOURCES += $(DEBIANDIR)/xe-linux-distribution.init
 SOURCES += $(DEBIANDIR)/xe-guest-utilities.postinst
-SOURCES += $(DEBIANDIR)/xe-guest-utilities.prerm
```

Comment:
why this file is excluded?

Reply:
There's only one command `rm /etc/apt/sources.list.d/citrix.list`, so I just removed this whole file.